src/circuit/constants.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+module.exports = {
+  PRIOIRY: 100,
```

Comment:
Let's take the priority out for now.

The way it will happen is:

When dialing to a peer:
1) try first all the transports that are direct connections
2) try any relay multiaddrs I know

We will always want direct connections if possible